modules/kubernetes/prometheus/templates/values-extras.yaml.tpl

```diff
@@ -11,3 +11,11 @@ volumeClaim:
   enabled: ${volume_claim_enabled}
   storageClassName: ${volume_claim_storage_class_name}
   size: ${volume_claim_size}
+
+resources:
+  requests:
+    memory: "250Mi"
+    cpu: "20m"
+  limits:
+    memory: "500Mi"
+    cpu: "100m"
```

Comment:
Lets not set cpu limits as it only artificially throttles CPU.

Reply:
Sorry old habits die hard